docs/api-reference/next/image.md

```diff
@@ -144,30 +116,32 @@ const MyImage = (props) => {
 
 ### sizes
 
-A string mapping media queries to device sizes. Defaults to `100vw`.
+A string which provides information about how wide the image will be at different breakpoints. Defaults to `100vw`--the full width of the screen.
```

Comment:
In this case I'm leaning a little bit toward "will" be. because setting the "sizes" prop is more like "I'm telling you that the image will be 45% of the width of the screen on mobile devices. Request from the srcset according"